Counterexamples/CliffordAlgebra_not_injective.lean

```diff
@@ -260,6 +264,16 @@ is not injective, as it sends the non-zero `α * β * γ` to zero. -/
 theorem algebraMap_not_injective : ¬Function.Injective (algebraMap K <| CliffordAlgebra Q) :=
   fun h => αβγ_ne_zero <| h <| by rw [algebraMap_αβγ_eq_zero, RingHom.map_zero]
 
+/-- Bonus counterexample: `Q` is a quadratic form that has no bilinear form. -/
+theorem Q_not_in_range_toQuadraticForm : Q ∉ Set.range BilinForm.toQuadraticForm := by
```

Comment:
It seems there is a positive result: if a quadratic form is induced by a bilinear form, then the algebraMap to the clifford algebra is injective.

Reply:
Correct; this is in fact a consequence of a (trivial) generalization of `equivExterior`; for now, we assume `Invertible (2 : R)` instead of `IsInducedByBilinear Q`, as the latter does not exist as a typeclass.